lib/sync_checker/checks/translations_check.rb

```diff
@@ -0,0 +1,32 @@
+module SyncChecker
+  module Checks
+    TranslationsCheck = Struct.new(:expected_locales) do
+      attr_reader :content_item
+      def call(response)
+        stringified_locales = expected_locales.map(&:to_s)
```

Comment:
I've removed the `.try` bit in `EditionBase` (in 99ef4be) as it is never nil. It shouldn't ever be `empty?` either. I think this is ok now and we can revise along they way if that proves not to be the case.